
[follow.py:154 2015-05-26 19:48:41,145] INFO run follow
[connectionpool.py:258 2015-05-26 19:48:41,182] INFO Starting new HTTP connection (1): yun.baidu.com
[connectionpool.py:375 2015-05-26 19:48:41,235] DEBUG Setting read timeout to None
[connectionpool.py:415 2015-05-26 19:48:41,310] DEBUG "GET /pcloud/friend/getfollowlist?query_uk=472662813&limit=24&start=0 HTTP/1.1" 200 None
[follow.py:171 2015-05-26 19:48:41,314] ERROR errno: -55
